index.html

```diff
@@ -51,4 +51,55 @@ <h2 id="img-caption"> &#9733 YES &#9733 WE &#9733 CAN &#9733</h2>
     <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
   </body>
 </html>
+<html>
+  <head>
+    <link rel="stylesheet" type="text/css" href="src/styles.css">
+    <meta charset="UTF-8" />
+    <title>RHCP</title>
+  </head>
+
+  <body>
+    <div id="main" >
+      <div id="app">
+        <h1 id="title">Barack Obama</h1>
+        <div id="img-div">
+          <img id="image" src ="https://observer.com/wp-content/uploads/sites/2/2016/12/obama_nyo_mosaic4-low.jpg?resize=768,896" alt="Trup members">
+          <h2 id="img-caption"> &#9733 YES &#9733 WE &#9733 CAN &#9733</h2>
+        </div>
+        <div id="tribute-info">
+            <p>Hope and change. With those words, eight years ago, America ushered in a remarkable period in our history. Barack Obama, a black man, won the presidency in a country where slavery and racism were incorporated into the nation’s Constitution and social fabric. That a man of mixed race and mixed religion who grew up in a single-parent household could be elected, re-elected, and serve ably and honorably, reflects greatness not only in the individual, but in a society that has come a very long way.</p>
+            
+            <p><strong>Obamas Dream,</strong> We have had disagreements with President Obama’s policies and politics. But upon reflection: he comported himself beyond reproach. There was never a hint of personal scandal. As a role model–as a father, as a man, as the elected head-of-state–he acted with conviction, with honesty, with compassion, and with intelligence. (How can you not admire a man who invites his mother-in-law to live him for eight years?) </p>
+        </div>
+        <p class="footer" >Find out more about the former president of america <a id="tribute-link" href="https://observer.com/2017/01/barack-obama-a-critics-appreciation/" target="_blank">here</a>!</p>
+      </div>
+    </div>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+  </body>
```

Comment:
Great job using polite words. I suggest that you add an explanation for this. As highlighted in the lesson[How to give a code review](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md)
"it might be obvious for you but not for someone else."
<img width="1654" alt="Screenshot 2022-10-31 at 12 05 31" src="https://user-images.githubusercontent.com/48546992/198994467-8dc71914-784c-4079-abdd-aa01193e30fe.png">